methods/coseismic/Coseismic_Requirement_Validation-Ridgecrest_asc.ipynb

```diff
@@ -2,7 +2,9 @@
  "cells": [
```

Comment:
<p>We should update this title. </p><p> </p><p>Also for part A and B lets do only 1 cell to remiove potentially errors challenges for reviewers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>